[2021-07-23 17:09:35,426] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.load_to_db 2021-07-23T14:09:30.620896+00:00 [queued]>
[2021-07-23 17:09:35,438] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.load_to_db 2021-07-23T14:09:30.620896+00:00 [queued]>
[2021-07-23 17:09:35,439] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-23 17:09:35,439] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-07-23 17:09:35,439] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-23 17:09:35,459] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): load_to_db> on 2021-07-23T14:09:30.620896+00:00
[2021-07-23 17:09:35,462] {standard_task_runner.py:52} INFO - Started process 24591 to run task
[2021-07-23 17:09:35,467] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'hello_world', 'load_to_db', '2021-07-23T14:09:30.620896+00:00', '--job-id', '55', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/tmp/tmpuobw8_e1', '--error-file', '/tmp/tmp6ikmmx7r']
[2021-07-23 17:09:35,468] {standard_task_runner.py:77} INFO - Job 55: Subtask load_to_db
[2021-07-23 17:09:35,518] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_world.load_to_db 2021-07-23T14:09:30.620896+00:00 [running]> on host DESKTOP-JHC8U0L.localdomain
[2021-07-23 17:09:35,593] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=hello_world
AIRFLOW_CTX_TASK_ID=load_to_db
AIRFLOW_CTX_EXECUTION_DATE=2021-07-23T14:09:30.620896+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-23T14:09:30.620896+00:00
[2021-07-23 17:09:35,594] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-23 17:09:35,596] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'python3 /home/dennismutia/vouchers/load_to_db.py']
[2021-07-23 17:09:35,603] {subprocess.py:74} INFO - Output:
[2021-07-23 17:09:36,334] {subprocess.py:82} INFO - Command exited with return code 0
[2021-07-23 17:09:36,377] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=hello_world, task_id=load_to_db, execution_date=20210723T140930, start_date=20210723T140935, end_date=20210723T140936
[2021-07-23 17:09:36,413] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-23 17:09:36,446] {local_task_job.py:149} INFO - Task exited with return code 0
